delfin/drivers/netapp/dataontap/netapp_handler.py

```diff
@@ -1246,3 +1246,28 @@ def get_capabilities(filters):
             NetAppHandler.get_cap_by_version(version, capabilities)
             cap_map[version] = capabilities
         return cap_map
+
+    def get_latest_perf_timestamp(self):
+        try:
+            timestamp = 0
+            json_info = self.do_rest_call(constant.CLUSTER_PERF_URL, None)
+            for perf_info in json_info:
+                occur_time = \
+                    int(time.mktime(time.strptime(
+                        perf_info.get('timestamp'),
+                        PerformanceHandler.TIME_TYPE))) * 1000
+                if timestamp < occur_time:
```

Comment:
The timestamp has an 8-hour time conversion. Please handle it.

Reply:
The modified